packages/Search/src/details/DTK_DetailsBox.hpp

```diff
@@ -8,8 +8,8 @@
  *                                                                          *
  * SPDX-License-Identifier: BSD-3-Clause                                    *
  ****************************************************************************/
-#ifndef DTK_Box_HPP
-#define DTK_Box_HPP
+#ifndef DTK_BOX_HPP
```

Comment:
Nice did you change it or did clang-format do it?

Reply:
I did it myself unfortunately :/